start-up/setting-up-reaction-roles.md

```diff
@@ -93,39 +75,40 @@ The `config` command offers a voice to edit group settings. Those are completely
+* `reverse`: When the message is reacted on, it will remove the role instead of assigning it, and vice versa.
+* `max-roles`: The maximum amount of roles within the group a user can asign themselves (0 = unrestricted).
+* `dm-notification`: When a role is assigned/removed, the bot will message the user.
+* `freeze`: When true, the group will not function.
+* `duration`: How long the user will keep their assigned role. This will *not* give a role back (in case of `reverse` being enabled).
 
 For an in-depth view of all options, see the [Configuring Reaction Roles](../admin-commands/config/configuring-reaction-roles.md) page.
 
 ## FAQ
 
 ### How do I make reaction roles temporary?
 
-Use `/config arguments: reactionroles <group> duration <duration>` to add a timer that starts when the reaction is clicked. The duration is provided in, say, [the same way as](arguments.md#durations) you use the `temprole` command.
+Use `/config reaction-roles <group> <duration>` to add a timer that starts when the reaction is clicked. The duration is provided in, say, [the same way as](arguments.md#durations) you use the `temprole` command.
 
 ### Can I make a single click of a reaction give out more than one role?
 
-Yes! However, group settings are common for each role in the group. Though you can apply multiple groups to the same message, which will allow you to:
+Yes! While group settings are common for each role in the group, you can apply multiple groups to the same message, which will allow you to:
 
-- Make some roles temporary, and other ones permanent
-- Make some roles only joinable, while removing other ones on click
+* Make some roles temporary, and other ones permanent
+* Make some roles only joinable, while removing other ones on click
 ... And much more, you can experiment which settings works best for you.
 
 ### How do I disable notifications that are sent to the user?
 
-#### If you are a server admin:
+#### If you are a server admin
 
-Use `/config arguments:reactionroles <group> dmnotification false` to disable notifications for a group.
+Use `/config reaction-roles <group> [dm-notification: false]` to disable notifications for a group.
 
-{% hint style="warning" %}
-If a message has multiple groups, notifications will still be sent, providing at least one of the other groups has notifications enabled
-{% endhint %}
+Doing this will only configure that specific group, so if there are multiple groups in the message, make sure to configure all of them to your liking.
 
-#### If you are an user:
+#### If you are a user
 
-You can disable all reaction role notifications in every server with `/userconfig arguments: reactionrolenotifications false` for your account. This will take precedence over server settings regardless.
+You can disable all reaction role notifications in every server with `/userconfig settings [reaction-role-notifications: false]` for your account. This will overwrite server settings regardless.
```

Comment:
Needs rewriting for clarity, in fact, only if user has this set to false will take precedence, also "overwrite" feels wrong to use here

Reply:
> You can disable all reaction role notifications with `/userconfig settings [reaction-role-notifications: false]`.